<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="CardWidget" name="CardWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,10,1">
        <item>
         <widget class="BodyLabel" name="BodyLabel">
          <property name="font">
           <font>
            <family>MiSans Medium</family>
            <pointsize>13</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>▌关于</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>MiSans</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="HyperlinkButton" name="HyperlinkButton_OpenSite">
            <property name="text">
             <string>官网</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="HyperlinkButton" name="HyperlinkButton_JoinGroup">
            <property name="text">
             <string>加入官方QQ群</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TransparentPushButton" name="TransparentPushButton_Github">
            <property name="text">
             <string>Github</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TransparentPushButton" name="TransparentPushButton_ColoredEgg">
            <property name="text">
             <string>??</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HyperlinkButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
